Decred Wallet/Wallet Utils/TransactionNotification.swift

```diff
@@ -73,8 +73,41 @@ class TransactionNotification: NSObject {
         
         if tx!.direction == DcrlibwalletTxDirectionReceived && WalletSettings(for: affectedWallet).txNotificationAlert != .none {
```

Comment:
https://github.com/planetdecred/dcrios/blob/a5c0545b222879aa37d17ec9905139d676762230/Decred%20Wallet/Wallet%20Utils/TransactionNotification.swift#L65-L67

on line 65 we already check to make sure it's not nil, at if it's nil we exit the function

so it's unlikely that would lead to a crash